reference/common_errors.rst

```diff
@@ -58,8 +49,7 @@ We can see this in action by distributing rows across worker nodes, and then run
 
 ::
 
-  ERROR:  40P01: canceling the transaction since it was involved in a distributed deadlock
-  LOCATION:  ProcessInterrupts, postgres.c:2988
+  ERROR:  canceling the transaction since it was involved in a distributed deadlock
```

Comment:
Has the error message changed to omit 40P01? If so, I wonder if there are other messages changed in a similar way...

Reply:
Probably it does. Will check it as well